ui/src/app/pages/admin/workspace/workspace-publishing-info.spec.tsx

```diff
@@ -137,7 +132,7 @@ describe('BasicInformation', () => {
 
   it('should disable publishing when workspace is locked  (enablePublishedWorkspacesViaDb = true)', async () => {
     enablePublishedWorkspacesViaDb();
-    workspace.featuredCategory = FeaturedWorkspaceCategory.COMMUNITY;
+    workspace.featuredCategory = undefined;
```

Comment:
update the test logic: an unpublished workspace would normally have an enabled publish button.  so this is additional confidence for the reason the button is disabled